lib/default-theme/NavLinks.vue

```diff
@@ -69,14 +69,22 @@ export default {
         })
       }))
     },
-    githubLink () {
+    repoLink () {
       const { repo } = this.$site.themeConfig
       if (repo) {
         return /^https?:/.test(repo)
           ? repo
           : `https://github.com/${repo}`
       }
-    }
+    },
+    repoLabel () {
+      if (this.$site.themeConfig.repoLabel) return this.$site.themeConfig.repoLabel
+
+      const repoHost = this.repoLink.match(/^https?:\/\/[^/]+/)[0] || 'github'
+      return ['GitHub', 'GitLab', 'Bitbucket'].find(platform => {
+        return repoHost.toLowerCase().includes(platform.toLowerCase())
```

Comment:
In client side you shouldn't use `includes` since Buble doesn't ployfill for that. 

BTW, why define a Uppercase constant and then `toLowerCase`?

Reply:
Oh okay, I would change this to `match` instead.

Defining uppercase constants since they are what we want to show on the header, lowercase since we want to compare. But actually using regex with case insensitive flag and match would be nicer.